zuul.d/layout.yaml

```diff
@@ -1,15 +1,23 @@
 ---
 # zuul.d/layout.yaml
+- job:
+    name: ansible-tox-devel
+    parent: ansible-tox-py36
+    vars:
+      tox_envlist: devel
+
+- job:
+    name: ansible-tox-distros
+    parent: ansible-tox-molecule
+    vars:
+      tox_envlist: distros
+    roles:
+      - zuul: install-docker
```

Comment:
This should be the project where install-docker is defined, so in this case opendev.org/zuul/zuul-jobs. However, we include this by default.

https://zuul-ci.org/docs/zuul/user/config.html#attr-job.roles.zuul